tests/unit_tests/test_oedisi_tools/test_metrics.py

```diff
@@ -0,0 +1,34 @@
+from oedisi.tools import cli
+import pytest
+from pathlib import Path
+from click.testing import CliRunner
+import numpy as np
+
+
+@pytest.fixture
+def base_path() -> Path:
+    """Get the current folder of the test"""
+    return Path(__file__).parent
+
+
+def test_metrics(base_path: Path, monkeypatch: pytest.MonkeyPatch):
+    monkeypatch.chdir(base_path)
+
+    runner = CliRunner()
+    result = runner.invoke(
+        cli, ["evaluate-estimate", "--path", Path("test_data"), "--metric", "MARE"]
+    )
+    assert result.exit_code == 0
+    assert np.isclose(float(result.output), 0.6281941853348966)
```

Comment:
Just checking - where are these hardcoded values taken from? Might there be differences based on the performance of the algorithm?

Reply:
This is running against the test data, which is very fixed.